feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/handling/BaseNotificationHandler.kt

```diff
@@ -47,12 +48,12 @@ abstract class BaseNotificationHandler(
     protected abstract suspend fun handleNotificationInternal(channelId: String, message: RemoteMessage): Boolean
 
     internal fun RemoteMessage.getMessageContent(): NotificationData {
-        val payload: Map<String, Any> = data["payload"]?.let { payload -> gson.fromJson(payload) } ?: emptyMap()
+        val payload: Map<String, Any?> = data["payload"]?.let { payload -> gson.fromJson(payload) } ?: emptyMap()
```

Comment:
Thats why we had an issue with "assetId: null" right?

Reply:
It seems that the main reason was the incorrect names of the notification types, and gson was already processing assetId: null.

I left this code just in case, but now it seems to me that it is superfluous

Reply:
Returned to the previous method